temporaltest/server.go

```diff
@@ -39,10 +40,43 @@ func (ts *TestServer) fatal(err error) {
 }
 
 // Worker registers and starts a Temporal worker on the specified task queue.
+// WorkflowPanicPolicy is set to worker.FailWorkflow
```

Comment:
Nit: I think it's probably ok to omit this information here, but keep it on the `NewWorkerWithOptions` and `WithBaseWorkerOptions` funcs.

```suggestion
```